.vsts-ci.yml

```diff
@@ -31,10 +31,10 @@ jobs:
         # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
         # Will eventually change this to two BYOC pools.
         ${{ if ne(variables['System.TeamProject'], 'internal') }}:
-          name: NetCorePublic-Int-Pool
+          name: VSEng-MicroBuildVS2019
```

Comment:
You want to change the pool to `NetCorePublic-Pool`
and then the queue to: `BuildPool.Windows.10.Amd64.VS2019.Open`